From 9793ad99a2496d6a94da1161cacd6492113beabb Mon Sep 17 00:00:00 2001
From: Stuart Mumford <stuart@cadair.com>
Date: Fri, 12 Jun 2020 21:04:43 +0100
Subject: [PATCH 1/2] Fix parfive requirements for 3.6

---
 setup.cfg | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 9f4e0b81b..94b30e888 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -39,7 +39,8 @@ install_requires =
   matplotlib>=2.2.2
   pandas>=0.23.0
   astropy>=3.2
-  parfive[ftp]>=1.1.0
+  parfive[ftp]>=1.1.0;python_version>="3.7"
+  parfive[ftp]>=1.0.2<1.1.0;python_version<="3.6"
 
 [options.extras_require]
 database = sqlalchemy
-- 
2.25.1

